pkg/arvo/sys/hoon.hoon

```diff
@@ -4217,12 +4217,20 @@
         $(inx +(inx))
     ==
   |-  ^-  ?
-  ?:  =(0 b)  &
-  =+  cur=(end 3 b)
+  ?:  =(inx len)  &
+  =+  cur=(cut 3 [inx 1] b)
   ?:  &((lth cur 32) !=(10 cur))  |
-  =+  len=(teff cur)
-  ?&  |(=(1 len) =+(i=1 |-(|(=(i len) &((gte (cut 3 [i 1] b) 128) $(i +(i)))))))
-      $(b (rsh [3 len] b))
+  =+  tef=(teff cur)
+  ?&  ?|
+          =(1 tef)
+          =+  i=1
+        |-  ?|
+            =(i tef)
+            ?&  (gte (cut 3 [(add i inx) 1] b) 128)
+                $(i +(i))
+        ==  ==
+      ==
+      $(inx +(inx))
```

Comment:
The indentation here is not idiomatic, it should look something like:

```
=+  tef=(teff cur)
?&  ?|  =(1 tef)
        =+  i=1
        |-  ^-  ?
        ?|  =(i tef)
            ?&  (gte (cut 3 [(add i inx) 1] b) 128)
                $(i +(i))
    ==  ==  ==
  $(inx +(inx))
==
```

Reply:
Ok, thanks. I was getting mixed up with the indent style for ?- and ?+. But shouldn't `$(inx +(inx))` be at the same indentation level as the `==  ==  ==`?

Reply:
Yeah, you're right.